moveit_ros/occupancy_map_monitor/src/occupancy_map_monitor_middleware_handle.cpp

```diff
@@ -81,6 +81,10 @@ OccupancyMapMonitorMiddlewareHandle::OccupancyMapMonitorMiddlewareHandle(const r
   if (parameters_.map_frame.empty())
   {
     node_->get_parameter("octomap_frame", parameters_.map_frame);
+    if (parameters_.map_frame.empty())
+    {
+      RCLCPP_ERROR(LOGGER, "No 'map_frame' defined for octomap updates");
```

Comment:
Should be `octomap_frame` .?
```suggestion
      RCLCPP_ERROR(LOGGER, "No 'octomap_frame' parameter defined for octomap updates");
```

Reply:
Yeah, good point. I guess that was changed since ROS1.